src/log.h

```diff
@@ -0,0 +1,31 @@
+/* SPDX-License-Identifier: ((GPL-2.0 WITH Linux-syscall-note) OR BSD-2-Clause) */
+/*
+ * Copyright (c) 2021 NVIDIA CORPORATION. All rights reserved
+ */
+#ifndef _LOG_H
+#define _LOG_H
+
+#include <arpa/inet.h>
+#include <stdio.h>
+
+#define IP2GID_SERVER_PORT 4791
```

Comment:
you are moving some defines that shouldn't be moved. for example: IP2GID_TIMEOUT_WAIT isn't log related.